app/src/main/java/org/fossasia/openevent/app/data/auth/EncryptionService.java

```diff
@@ -0,0 +1,12 @@
+package org.fossasia.openevent.app.data.auth;
+
+import org.fossasia.openevent.app.data.auth.model.Login;
+
+public interface EncryptionService {
```

Comment:
Encryption Service would do encryption instead of getting email and password

Reply:
In presenter/VM

You still haven't created a model for encryption

Reply:
Oh Sorry. I thought that using the Login model will do as the new encryption model will have the same instance variables (email and password) as the login model

Reply:
The encryption service will not need context at all in its functions. It will be an interface, its implementation will encapsulate context. As mentioned earlier, check preference class mentioned above in conversation with @srv-twry

Reply:
@srv-twry If you were able to get what I was trying to say, can you elaborate the idea and collaborate with @sridharjajoo

Reply:
@iamareebjamal I get your point.